### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by Rashdan98**
ready

### CODE REVIEW

**Comment by Rashdan98** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.1.4'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```
Comment (ngocvt-0484) [inline Gemfile]: please use gem "mysql2"

Comment (ngocvt-0484) [reply Gemfile]: delete sqlite3

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+
+development:
+  <<: *default
```
Comment (ngocvt-0484) [inline config/database.yml]: this file will not be up to GIt, please
```
git rm --cached config/database.yml
```
and in .gitignore add
```
config/database.yml
```

File: Gemfile (hunk 1 above)
Comment (Rashdan98) [reply Gemfile]: how to do this?

Comment (Rashdan98) [reply Gemfile]: do i need to install mysql2 in terminal?

**Comment by Rashdan98** [comment]
ready

**Comment by Rashdan98** [comment]
ready

**Comment by Rashdan98** [comment]
ready

File: config/database.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
```
Comment (huongnt-2545) [inline config/database.yml]: apply config database using MySQL DB, instead of SQLite3

**Comment by Rashdan98** [comment]
ready

File: config/database.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```
Comment (huongnt-2545) [inline config/database.yml]: Refer the sample for database.yml and apply it (note: replace to your DB `username` and `password`)
refs: 
https://edgeguides.rubyonrails.org/configuring.html#configuring-a-mysql-or-mariadb-database
https://gist.github.com/jwo/4512764#file-mysql-database-yml

**Comment by Rashdan98** [comment]
ready

**Comment by Rashdan98** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged
You can merge this pull request

**Pull request merged**